packages/marketplace/src/resources/snippets.ts

```diff
@@ -155,4 +155,10 @@ export default [
     "code": `.main-collectionLinkButton-collectionLinkButton[href="/collection/tracks"], .main-collectionLinkButton-collectionLinkButton[href="/collection/episodes"] {display: none;}`,
     "preview": "resources/assets/snippets/remove-ep-likes.png",
   },
+  {
+    "title": "Vinyl Cover Art",
```

Comment:
Maybe change the name to "Rotating Cover Art", since that's what's in the preview image?

Reply:
Ye changed it! I also fixed linting error. Thanks! <3

Reply:
Thanks :) Still missing a comma at the end of the preview line

Reply:
Fixed!

Reply:
👌